docs/api/mount/hasAttribute.md

```diff
@@ -1,17 +1,14 @@
-# hasAttribute(attribute, value)
```

Comment:
Hummm... I think it's better to encourage people to use `getAttribute()` which is competent to do the same job and does it better :]